www-servers/cherokee/cherokee-1.2.104.recipe

```diff
@@ -0,0 +1,122 @@
+SUMMARY="A high-performance web server"
+DESCRIPTION="Cherokee is a high-performance web server. It is very fast, \
+flexible and easy to configure. It offers support for the widespread \
+technologies nowadays: FastCGI, SCGI, PHP, CGI, SSI, TLS and SSL \
+encrypted connections, Virtual hosts, Authentication, on the fly encoding, \
+Load Balancing, Apache compatible log files, Data Base Balancer, \
+downtime-free updates and upgrades, Reverse HTTP Proxy, and much more."
+HOMEPAGE="http://www.cherokee-project.com"
+COPYRIGHT="2001-2011 Alvaro Lopez Ortega"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/cherokee/webserver/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="5cbd00ff48503eaf90356b2975e311c02977f9166927e57fc23f541a109efd98"
+SOURCE_DIR="webserver-$portVersion"
+PATCHES="cherokee-$portVersion.patchset"
+
+GLOBAL_WRITABLE_FILES="
+	settings/cherokee/cherokee.conf.perf_sample keep-old
+	settings/cherokee/cherokee.conf keep-old
+	var/www/images/cherokee-logo.png keep-old
+	var/www/images/default-bg.png keep-old
+	var/www/images/favicon.ico keep-old
+	var/www/images/powered_by_cherokee.png keep-old
+	var/www/index.html keep-old
+	"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
I'd drop x86_gcc2 if it works for x86 as secondary arch. Or remove the secondary arch.
The libraries are only useful for the arch running the server.